libs/application/templates/driving-school-confirmation/src/fields/ConfirmationDate/index.tsx

```diff
@@ -1,15 +1,22 @@
 import React, { FC } from 'react'
+import format from 'date-fns/format'
 import { GridContainer, GridRow, GridColumn } from '@island.is/island-ui/core'
 import { m } from '../../lib/messages'
 import { useLocale } from '@island.is/localization'
 import { FieldBaseProps } from '@island.is/application/types'
 import { DatePickerController } from '@island.is/shared/form-fields'
 import { useFormContext } from 'react-hook-form'
 
+const df = 'yyyy-MM-dd'
+
 const ConfirmationDate: FC<FieldBaseProps> = ({ field, error }) => {
   const { id } = field
   const { formatMessage } = useLocale()
   const { setValue } = useFormContext()
+  const defaultDate = new Date()
+  // Set date to yesterday if current time is before noon for better UX
```

Comment:
Not a review comment. Just curious how this was established.
User testing or just general feedback?